src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/FirstLastN.cs

```diff
@@ -66,5 +66,32 @@ public override bool CheckTypes(CheckTypesContext context, TexlNode[] args, DTyp
             return fArgsValid;
         }
     }
+
+    // FirstN(source:*, [count:n])
+    // LastN(source:*, [count:n])
+    internal sealed class FirstLastNFunction_UO : BuiltinFunction
+    {
+        public override bool IsSelfContained => true;
+
+        public override bool SupportsParamCoercion => false;
```

Comment:
>false [](http://example.com/codeflow?start=54&length=5)

We should support coercion on the second parameter - strings should be coercible to numbers, for example.

Reply:
Oh, I see that the "regular" FirstN/LastN also don't support coercion. We should fix that - I'll send a PR soon

Reply:
Implemented this and added tests.